libstorage-ng.spec.in

```diff
@@ -89,6 +89,7 @@ Obsoletes:      libstorage %(echo `seq -s " " -f "libstorage%.f" 9`)
 %ifarch s390 s390x
 Requires:       s390-tools
 %endif
+Conflicts:      libdmraid.so.1.0.0.rc16(libdmraid.so.1.0.0.rc16)(64bit)
```

Comment:
Shouldn't we have a comment / bug reference?

Reply:
Yes, we should (ideally point to the comment: https://bugzilla.suse.com/show_bug.cgi?id=1088570#c35), otherwise LGTM.

Note: I tried Obsoletes to push towards automatic conflict resolution, but it did not work. Need of manual intervention is acceptable, though.